search/typesense.py

```diff
@@ -71,8 +72,18 @@
         "query_fields": ["course", "org", "number", "content"],
     },
     COURSE_CONTENT_INDEX: {
+        # Most fields will be auto-created but some (datetimes, facet fields) need to be specified up front:
         "field_overrides": [
             {"name": "start_date", "type": "datetime", "optional": True},
+            # Enable stemming for the "content" field, so that e.g.
+            # searching for "run" will match "running", "runs", "ran".
+            # Unfortunately, this could break indexing if non-string fields get
+            # included in "content", which is possible (XBlocks can return
+            # whatever data they want from their `index_dictionary()` method).
```

Comment:
Can we convert all fields in `content` to a string before indexing to avoid this?